pkg/service/group.go

```diff
@@ -204,6 +207,14 @@ func (s *groupService) createGroupRoleRelations(ctx context.Context, db bun.IDB,
 			}
 
 			namespace := pnr.GetNamespace()
+			match := regexc.MatchString(namespace)
```

Comment:
trying to understand what exactly we are doing here ?

Reply:
As per the latest regex, if doesn't match, the namespace name is invalid. `MatchString` returns boolean